Sources/App/Views/Common/Buttons/AccentButtonStyle.swift

```diff
@@ -2,23 +2,22 @@
 //  Mocka
 //
 
+import AppKit
 import SwiftUI
 
 /// This button adheres to `ButtonStyle` protocol set the button color to the `accentColor` selected by the user.
 struct AccentButtonStyle: ButtonStyle {
+  /// The color scheme of the app.
+  let colorScheme: ColorScheme
+
   /// Whether or not the button is enabled.
   let isEnabled: Bool
 
-  func makeBody(configuration: Self.Configuration) -> some View {
-    configuration.label
-      .foregroundColor(configuration.isPressed ? Color.accentColor : Color.latte)
-      .background(backgroundColor(isPressed: configuration.isPressed))
-      .cornerRadius(6.0)
-  }
-
   /// Returns an instance of `AccentButtonStyle`
+  /// - Parameter colorScheme: The color scheme of the app.
   /// - Parameter isEnabled: Whether or not the button is enabled.
-  init(isEnabled: Bool = true) {
+  init(colorScheme: ColorScheme, isEnabled: Bool = true) {
+    self.colorScheme = colorScheme
```

Comment:
Is this really necessary?
Can you access to `@Environment(\.colorScheme)` from here?

Reply:
I can't because it wouldn't be injected in `AccentButtonStyle` as it is not a `View`